frontend/src/framework/services/notification-service/enums/notification-status.enums.ts

```diff
@@ -0,0 +1,9 @@
+enum NotificationStatus {
+    Info = 'info',
+    Success = 'success',
+    Warning = 'warning',
+    Error = 'error',
+    Default = 'default',
+}
+
+export { NotificationStatus };
```

Comment:
Replace with `as const` enum, keys should be uppercased. Also I would suggest rename `NotificationStatus` to `NotificationType` for clarity.